service/create/validation_test.go

```diff
@@ -0,0 +1,69 @@
+package create
+
+import (
+	"testing"
+
+	"github.com/giantswarm/awstpr/aws"
+	"github.com/juju/errgo"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidateWorkers(t *testing.T) {
+	tests := []struct {
+		name          string
+		workers       []aws.Node
+		expectedError error
+	}{
+		{
+			name: "Valid workers - image IDs and instance types are the same",
+			workers: []aws.Node{
+				aws.Node{
+					ImageID:      "example-image-id",
+					InstanceType: "example-instance-type",
+				},
+				aws.Node{
+					ImageID:      "example-image-id",
+					InstanceType: "example-instance-type",
+				},
+			},
```

Comment:
I would like to always define all properties of the test case in here to make it more clear of what is going on. Here I would like to add `expectedError: nil,`.